src/main/java/data/domain/RacingGame.java

```diff
@@ -0,0 +1,41 @@
+package data.domain;
+
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class RacingGame {
+
+    private static final int END_GAME_LAPS = 0;
+    private static final int ENGINE_MAX_POWER = 10;
+    private static final Random RANDOM = new Random();
+
+    private List<Car> cars;
+    private int laps;
+
+    public RacingGame(int participants, int laps) {
+        this.cars = ready(participants);
+        this.laps = laps;
+    }
+
+    private List<Car> ready(int participants) {
+        return Stream.generate(Car::new)
+            .limit(participants)
+            .collect(Collectors.toList());
+    }
+
+    public int [] move() {
```

Comment:
```suggestion
    public List<Integer> move() {
```

이펙티브 자바 `아이템 28. 배열보다는 리스트를 사용하라`를 참고해보면 어떨까요?

Reply:
최대한 심플하게 만들어보고 싶었는데, 사소한 것도 잘 챙겨야 견고한 소프트웨어를 만들 수 있겠죠! 🙇